src/kaleidoscope/Hardware.h

```diff
@@ -64,6 +72,16 @@ class Hardware {
    * changes made before this call are reflected on the device.
    */
   void syncLeds(void) {}
+  /**
+   * Set the color of a per-key LED at a given row and column.
+   *
+   * Setting the color does not need to take effect immediately, it can be
+   * delayed until @ref syncLeds is called.
+   *
+   * @param ledAddr is the matrix address of the LED.
+   * @param color is the color to set the LED to.
+   */
+  void setCrgbAt(LEDAddr ledAddr, cRGB color) {}
```

Comment:
Style guide again: `led_addr`, not `ledAddr` for variables.

Reply:
Thanks, I'll fix that.

Reply:
Fixed.